modules/identity-provider-htpasswd-secret.adoc

```diff
@@ -14,14 +14,27 @@ contains the HTPasswd user file.
 
 .Procedure
 
-* Create an {product-title} `Secret` object that contains the HTPasswd users file.
+* Create a `Secret` object that contains the HTPasswd users file:
 +
 [source,terminal]
 ----
-$ oc create secret generic htpass-secret --from-file=htpasswd=</path/to/users.htpasswd> -n openshift-config
+$ oc create secret generic htpass-secret --from-file=htpasswd=<path_to_users.htpasswd> -n openshift-config <1>
 ----
+<1> The secret key containing the users file for the `--from-file` argument must be named `htpasswd`, as shown in the above command.
 +
-[NOTE]
+[TIP]
 ====
-The secret key containing the users file for the `--from-file` argument must be named `htpasswd`, as shown in the above command.
+You can alternatively apply the following YAML to create the secret:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: htpass-secret
+  namespace: openshift-config
+type: Opaque
+data:
+  htpasswd: <base64_encoded_htpasswd_file_contents>
+----
```

Comment:
when use the yaml file to create the secret, if there is secret with the same name already exists, it can be created successfully with no error prompted, while fail for oc commands for this case.